proto/cosmos/app/v1alpha1/config.proto

```diff
@@ -14,6 +14,14 @@ import "google/protobuf/any.proto";
 message Config {
   // modules are the module configurations for the app.
   repeated ModuleConfig modules = 1;
+
+  // golang_bindings specifies explicit interface to implementation type bindings which
+  // depinject uses to resolve interface inputs to provider functions.  The scope of this
+  // field's configuration is global (not module specific) and should be of the form:
+  //
+  // golang_bindings:
+  //   interfaceType: implementationType
+  map<string, string> golang_bindings = 2;
```

Comment:
Not sure we consider maps in proto messages kosher yet. We can serialize them deterministically, but I think the assumption is still that we shouldn't use them